src/main/kotlin/uk/co/poolefoundries/baldinggate/input/EditorInputHandler.kt

```diff
@@ -0,0 +1,130 @@
+package uk.co.poolefoundries.baldinggate.input
+
+
+import com.badlogic.gdx.Input.Buttons
+import com.badlogic.gdx.Input.Keys
+import com.badlogic.gdx.InputProcessor
+
+interface EditorInputHandler {
+    fun pause()
+    fun leftClick(x: Int, y: Int)
+    fun rightClick(x: Int, y: Int)
+    fun dragCamera(deltaX: Float, deltaY: Float)
+    fun zoom(amount: Int)
+}
+
+object RawEditorInputHandler : InputProcessor {
+
+    private var lastX = 0f
+    private var lastY = 0f
+    private var middle = false
+    private var left = false
+    private var right = false
+    private val listener = LevelEditorInputProcessor
+    // todo get list of valid actions that aren't movement and display on UI
```

Comment:
TODO comments should be in the format `// TODO (person/ticket reference): some summary of the thing to do`